addons/mail/static/src/models/clock_watcher/clock_watcher.js

```diff
@@ -0,0 +1,22 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one } from '@mail/model/model_field';
+
+/**
+ * Models a record that makes use of a clock.
+ */
+registerModel({
+    name: 'ClockWatcher',
+    identifyingFields: [['activityView']],
+    fields: {
+        activityView: one('ActivityView', {
```

Comment:
I would suggest `activityViewOwner` as name to follow our guideline to naming "parent" identifying fields.